Oliver-ke/Teamwork
# #28594864 Setup Babel and Eslint

#### What does this PR do?
- Enables the use of es6 feature and linting

#### Description of Task to be completed?
- add require packages
- add configuration files

#### How should this be manually tested?
- clone this repository
- run the server using `npm run start:dev`
- navigate your browser to `localhost:5000/api/v1`
- you should see a welcome message

#### Any background context you want to provide?
- none

#### What are the relevant project board stories?
[#28594864](https://github.com/Oliver-ke/Teamwork/projects/1#card-28594727)

#### Screenshots (if appropriate)
- none
#### Questions:
- none
